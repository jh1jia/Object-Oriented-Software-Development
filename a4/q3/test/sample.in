sample.txt doublewords dropfirst 2 count l
sample.txt allcaps
