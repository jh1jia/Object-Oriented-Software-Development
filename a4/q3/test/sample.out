12o
34o
r5d
r6d
HELLO
WORLD
